[2024-07-18 06:35:21,315] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: xcom_example_dag.pull_task 2024-03-14T00:00:00+00:00 [queued]>
[2024-07-18 06:35:21,438] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: xcom_example_dag.pull_task 2024-03-14T00:00:00+00:00 [queued]>
[2024-07-18 06:35:21,439] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-07-18 06:35:21,439] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2024-07-18 06:35:21,440] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-07-18 06:35:21,532] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): pull_task> on 2024-03-14T00:00:00+00:00
[2024-07-18 06:35:21,540] {standard_task_runner.py:52} INFO - Started process 2990 to run task
[2024-07-18 06:35:21,551] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'xcom_example_dag', 'pull_task', '2024-03-14T00:00:00+00:00', '--job-id', '507', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/xcom.py', '--cfg-path', '/tmp/tmpywnh6elb', '--error-file', '/tmp/tmpfbkal_zq']
[2024-07-18 06:35:21,566] {standard_task_runner.py:77} INFO - Job 507: Subtask pull_task
[2024-07-18 06:35:21,856] {logging_mixin.py:104} INFO - Running <TaskInstance: xcom_example_dag.pull_task 2024-03-14T00:00:00+00:00 [running]> on host a868c7bacc12
[2024-07-18 06:35:22,195] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=xcom_example_dag
AIRFLOW_CTX_TASK_ID=pull_task
AIRFLOW_CTX_EXECUTION_DATE=2024-03-14T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-14T00:00:00+00:00
[2024-07-18 06:35:22,238] {logging_mixin.py:104} INFO - Pulled message: Hello from push_function!
[2024-07-18 06:35:22,245] {python.py:118} INFO - Done. Returned value was: None
[2024-07-18 06:35:22,280] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=xcom_example_dag, task_id=pull_task, execution_date=20240314T000000, start_date=20240718T063521, end_date=20240718T063522
[2024-07-18 06:35:22,446] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-18 06:35:22,478] {local_task_job.py:146} INFO - Task exited with return code 0
